pr_agent/servers/github_app.py

```diff
@@ -203,8 +203,9 @@ def _check_pull_request_event(action: str, body: dict, log_context: dict, bot_us
     return pull_request, api_url
 
 
-async def _perform_commands(commands: List[str], agent: PRAgent, body: dict, api_url: str, log_context: dict):
+async def _perform_commands(agent: PRAgent, body: dict, api_url: str, log_context: dict):
     apply_repo_settings(api_url)
+    commands = get_settings().github_app.pr_commands
```

Comment:
The commands list for push trigger should be taken from the `push_commands` configuration - this change will cause the push trigger to perform the same commands list as when opening a PR, instead of its own separate list of commands.

I think a simple fix for this could be to revert this is to change the `_perform_commands` method to receive not the list of commands, but the name of the configuration from which to read that list, and the we can get that list after applying repo settings.

~But I wonder, why not just run `apply_repo_settings` right at the start? All the way back when we receive the request?~ Ok scratch that, we need to find the `api_url` first...